code/modules/martial_arts/cqc.dm

```diff
@@ -0,0 +1,186 @@
+#define SLAM_COMBO "HG"
+#define KICK_COMBO "HH"
+#define RESTRAIN_COMBO "GG"
+#define PRESSURE_COMBO "DG"
+#define CONSECUTIVE_COMBO "DDH"
+
+/datum/martial_art/cqc
+	name = "CQC"
+	help_verb = /mob/living/carbon/human/proc/CQC_help
+	block_chance = 75
+
+/datum/martial_art/cqc/proc/drop_restraining()
+	restraining = FALSE
+
+/datum/martial_art/cqc/proc/check_streak(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	if(findtext(streak,SLAM_COMBO))
```

Comment:
As a rule, Always space between the comma and the next arg in the list..Even here.